bindings/Makefile

```diff
@@ -16,12 +17,23 @@ deps/$(CODEGEN_BIN):
 build/%: $(SWAGGER_SPEC) deps/$(CODEGEN_BIN)
 	mkdir -p build/$*
 	java -jar deps/$(CODEGEN_BIN) generate -i $(SWAGGER_SPEC) \
-		-l $* -o build/$*
+		-l $* -o build/$* -c config.json
 	touch build/$*
+	if [ $* = "python" ] ; then \
+		rm -rf $(CLIENT_PATH); \
+		mkdir -p $(CLIENT_PATH)\api; \
+		mkdir -p $(CLIENT_PATH)\models; \
+		cp -rf ./build/python/determined._swagger.client/ $(CLIENT_PATH); \
+		cp -rf ./build/python/determined/_swagger/client/ $(CLIENT_PATH); \
+	fi
 
 .PHONY: build
 build: build/typescript-fetch build/python
 
+.PHONY: check
+check:
```

Comment:
This `check` target is literally always going to pass in CI.  You have to force regeneration to happen for it to be a meaningful check.  Please see how the `check-gen` target in `master/Makefile` does it.